README.rst

```diff
@@ -1,3 +1,7 @@
+.. image:: doc/_static/dipy-banner.png
+     :alt: DIPY - Diffusion Imaging in Python
```

Comment:
I do not why but it seems `:target:` should be before `:alt:` otherwise we do not see the image. Can you inverse this 2 properties?
Thanks @jhlegarreta

Reply:
Thanks for the review. Done !